test/declaration/class.unit

```diff
@@ -73,4 +73,12 @@ abstract base mixin class C13 {}
 class SomeClass native "Zapp" {
 }
 <<<
-class SomeClass native "Zapp" {}
\ No newline at end of file
+class SomeClass native "Zapp" {}
+>>> Record type with multiple fields in parameter has no trailing comma.
+class C { C((TypeName,TypeName,) this.record) {;} }
+<<<
+class C {
+  C((TypeName, TypeName) this.record) {
```

Comment:
Is this mostly testing that we can handle record types on initializing formals? If it's just about using a record type in a parameter, you could make a simpler example like:

```dart
function((TypeName, TypeName) parameter) { ; }
```

(And put it in probably "type/function.stmt".)

Reply:
Moved that and a bunch of other tests to `type/function.stmt`.